src/__snapshots__/router.test.ts.snap

```diff
@@ -1,11 +1,11 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
I migrated this repo from jest to vitest to avoid ESM import issues. It was really straightforward to do and didn't create many changes. Its also faster 🔥.